nav.go

```diff
@@ -457,7 +459,28 @@ func (nav *nav) position() {
 	}
 }
 
-func (nav *nav) preview(path string, win *win) {
+func (nav *nav) previewLoop(ui *ui) {
+	var path string
+	for {
+		p, ok := <-nav.previewChan
+		if !ok {
+			return
+		}
+		if len(p) != 0 {
+			win := ui.wins[len(ui.wins)-1]
+			nav.previewGen(p, win)
+			path = p
+		} else if len(gOpts.cleaner) != 0 && len(gOpts.cleaner) != 0 && nav.volatilePreview {
+			cmd := exec.Command(gOpts.cleaner, path)
```

Comment:
Isn't `path` empty here?

Reply:
It shouldn't be, since nav.volatilePreview is part of the condition of the second branch, and that variable only gets set during calls to previewGen (which is is only called within previewLoop). So, the second branch of that if statement will only be evaluated after iterations where previewGen is called and the path is set.